src/Illuminate/Foundation/Console/Kernel.php

```diff
@@ -294,6 +299,33 @@ protected function bootstrappers()
     }
 
     /**
+     * Set environment if 'env' option if provided.
+     *
+     * @param  InputInterface  $input
+     * @return void
+     */
+    protected function setEnvironment($input)
+    {
+        if (! app()->runningInConsole()) {
```

Comment:
@GrahamCampbell in this file `$this->app` implements the `Illuminate\Contracts\Foundation\Application` contract which doesn't have `environmentFile`, `loadEnvironmentFrom` functions. That's why i have used `app()` global helper here, so the `--env` option can be easily loaded.